cobalt/BUILD.gn

```diff
@@ -81,8 +81,5 @@ group("default") {
         "//components/update_client",
       ]
     }
-    if (!build_with_separate_cobalt_toolchain) {
-      deps += [ "//third_party/llvm-project/libunwind:unwind_evergreen" ]
```

Comment:
Where does this dep now come from?

Reply:
I guess it comes from starboard—I'm surprised that doesn't cause dependency check issues (is it in public_deps?), but this seems fine